DAO.sol

```diff
@@ -821,7 +822,10 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        // this can only be called after `quorumHalvingPeriod` has passed or at anytime
+        // by the curator with a delay of at least `minProposalDebatePeriod` between the calls
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now - minProposalDebatePeriod)) {
```

Comment:
for 1.) your address has to be on whitelist. If a non-sense address makes it to the whitelist, then this is an opportunity to split.

So the attack is only possible if a malicious address is on the whitelist. In the case, by accident, the curator sets such a n address on the whitelist, and then you see 1000 proposals to him, then the curator should remove him from the list, if he does not, then you have malicious curator, then you have to split.